From de185e36110efb131ffbec8cf337de82aabf0d93 Mon Sep 17 00:00:00 2001
From: Tobias Thierer <tobiast@google.com>
Date: Thu, 21 Apr 2016 12:03:33 +0100
Subject: [PATCH] fwb: Avoid needless Integer.valueOf() object allocation

Replaced call to method taking Object with call that takes int.

Bug: 28289401
Change-Id: I570ed9d40ca6239fab33355de2babefd509db9aa
---
 core/java/android/view/ViewDebug.java                           | 2 +-
 .../core/java/com/android/server/tv/TvInputManagerService.java  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/java/android/view/ViewDebug.java b/core/java/android/view/ViewDebug.java
index 05970aeb52aae..17356d04f64ef 100644
--- a/core/java/android/view/ViewDebug.java
+++ b/core/java/android/view/ViewDebug.java
@@ -1929,7 +1929,7 @@ public static void setLayoutParameter(final View view, final String param, final
                     + param + " is of type " + f.getType().getSimpleName());
         }
 
-        f.set(p, Integer.valueOf(value));
+        f.setInt(p, value);
 
         view.post(new Runnable() {
             @Override
diff --git a/services/core/java/com/android/server/tv/TvInputManagerService.java b/services/core/java/com/android/server/tv/TvInputManagerService.java
index e000d6f45f767..c7324c4ad34b8 100644
--- a/services/core/java/com/android/server/tv/TvInputManagerService.java
+++ b/services/core/java/com/android/server/tv/TvInputManagerService.java
@@ -3110,7 +3110,7 @@ protected void dump(FileDescriptor fd, final PrintWriter writer, String[] args)
                 pw.increaseIndent();
                 for (int i = 0; i < mUserStates.size(); i++) {
                     int userId = mUserStates.keyAt(i);
-                    pw.println(Integer.valueOf(userId));
+                    pw.println(userId);
                 }
                 pw.decreaseIndent();
 
